index.html

```diff
@@ -982,10 +974,23 @@ <h3 property="schema:name skos:prefLabel">Terminology</h3>
                     property="skos:prefLabel"
                     typeof="skos:Concept"
                   >
-                    <dfn id="extended-profile">Solid Profile</dfn>
+                    <dfn id="extended-profile">Extended Profile</dfn>
                   </dt>
                   <dd about="#extended-profile" property="skos:definition">
-                    An <dfn>extended profile</dfn> is...
+                    An <dfn>extended profile</dfn> is an additional profile document that provides supplemental description to a Solid Profile to help distribute and manage information about a WebID for use in different circumstances. Extended Profile Documents can be used for various purposes, such as organizing information or limiting access to certain data.
+                  </dd>
+                </dl>
+
+                <dl>
+                  <dt
+                    about="#preferences-document"
+                    property="skos:prefLabel"
+                    typeof="skos:Concept"
+                  >
+                    <dfn id="preferences-document">Preferences Document</dfn>
+                  </dt>
+                  <dd about="#preferences-document" property="skos:definition">
+                    A <dfn>preferences document</dfn>  is a specific resource intended to store data which helps authorized applications accommodate the user's preferences, such as settings, personalization preferences, and pointers to the user's private data.
```

Comment:
I forget the dfn syntax that lets you have multiple spellings/phrasings, so `extended profile` may be treated as equivalent to `extended Solid profile`... which I think would be optimal.
```suggestion
                    An <dfn>extended Solid profile</dfn> is an additional profile document that provides supplemental description to a Solid profile to help distribute and manage information about a WebID for use in different circumstances. Extended profile documents can be used for various purposes, such as organizing information or limiting access to certain data.
                  </dd>
                </dl>

                <dl>
                  <dt
                    about="#preferences-document"
                    property="skos:prefLabel"
                    typeof="skos:Concept"
                  >
                    <dfn id="preferences-document">Preferences Document</dfn>
                  </dt>
                  <dd about="#preferences-document" property="skos:definition">
                    A <dfn>preferences document</dfn> is a specific resource intended to store data which helps authorized applications accommodate the user's preferences, such as settings, personalization details, and pointers to the user's private data.
```